### PR TITLE
USWDS 3.0 - Copy edits

### PR DESCRIPTION
**Pull request by amyleadem**
## Description

Final proofread of the 3.0 docs

Proofread the following pages:
- [x] Intro post
- [x] Packages page
- [ ] Migration guide
- [ ] Getting started for devs
- [ ] USWDS Readme

### CODE REVIEW

File: _posts/2022-04-28-introducing-uswds-3-0.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ USWDS 3.0 stylesheets use the most modern version of the Sass module syntax. The
 
 Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
 
-[Read more about USWDS Packages]({{ site.baseurl }}/components/packages/) →
+[Read more about USWDS packages]({{ site.baseurl }}/components/packages/) →
 
 ### Simplify your project settings
```
Comment (amyleadem) [inline _posts/2022-04-28-introducing-uswds-3-0.md:42]: @thisisdano Feeling like we need to update the [Settings page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/develop/documentation/settings/) for 3.0, and linking to it in this section might cause confusion. For today, would it make sense to reroute the link to the [Sass modules section of the migration guide ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/develop/documentation/migration/#5-update-to-sass-module-syntax)?

**Comment by amyleadem** [comment]
Another quiz for you @thisisdano 
cc: @mejiaj 
I compared the list of items in the [packages table on the package page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/develop/components/packages/#included-packages) with the items in the generated .tgz package. I noticed the following packages were not included in the package table. Will you check which items you want included in that table? (Please note the all/none options at the bottom)

- [ ] _usa-password
- [ ] templates
- [ ] Usa-add-aspect
- [ ] Usa-fonts
- [ ] Usa-input-list
- [ ] Usa-type-line-length
- [ ] Usa-type-setting
- [ ] Usa-type-spacing
- [ ] Uswds-core
- [ ] Uswds-form-templates
- [ ] Uswds-tokens
- [ ] Uswds-validation
- [ ] All of the above
- [ ] None of the above

File: pages/ui-components/packages.md [hunk 2]
```diff
@@ -123,10 +127,12 @@ Using only package source would result in the following:
 @forward "usa-accordion/src/styles";
 @forward "usa-banner/src/styles";
 @forward "usa-button/src/styles";
+@forward "usa-icon/src/styles";
```
Comment (amyleadem) [inline pages/ui-components/packages.md:130]: usa-accordion/_index.scss had a dependency of usa-icon - assumed we wanted to point to the src/styles here

File: pages/ui-components/packages.md [hunk 3]
```diff
@@ -123,10 +127,12 @@ Using only package source would result in the following:
...
 @forward "usa-identifier/src/styles";
 @forward "usa-layout-grid/src/styles";
 @forward "usa-list/src/styles";
 @forward "usa-media-block/src/styles";
+@forward "uswds-core";
```
Comment (amyleadem) [inline pages/ui-components/packages.md:135]: usa-identifier/_index.scss has a dependency of uswds-core

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 1 above)
Comment (thisisdano) [reply _posts/2022-04-28-introducing-uswds-3-0.md:42]: Made updates to settings page here: https://github.com/uswds/uswds-site/pull/1564

**Pull request closed without merging**